app/components/post_list/post_list.test.tsx

```diff
@@ -68,7 +66,7 @@ describe('components/post_list/PostList', () => {
         mockPostModel(),
     ];
 
-    const baseProps: typeof PostListProps = {
+    const baseProps: typeof exportedForTesting.PostListProps = {
```

Comment:
You can use `ComponentProps<typeof PostList>` and will get the same result. That way you can remove also the export on `post_list.tsx`.